api/queries_comments.go

```diff
@@ -99,3 +100,30 @@ func CommentsForPullRequest(client *Client, repo ghrepo.Interface, pr *PullReque
 
 	return &Comments{Nodes: comments, TotalCount: len(comments)}, nil
 }
+
+func CommentCreate(client *Client, repoHost string, params map[string]string) (string, error) {
```

Comment:
We've had the pattern of accepting a `map[string]string` params object elsewhere, but ideally we shouldn't use it too much. Instead, I would suggest accepting a struct. That way, the caller is certain that is passing only the fields that are actually supported.